Makefile

```diff
@@ -59,7 +59,7 @@ setup_test: cl.exe
 # smoke test - can the wrapper compile anything
 build_and_check_test_sample : setup_test
 	cd tmp\test
-	cl /c /EHsc ..\..\test\calc.cxx /DCALC_EXPORTS /I ..\..\test\include
+	cl /c /EHsc "..\..\test\src file\calc.cxx" /DCALC_EXPORTS /DCALC_HEADER=\"calc.h\" /I ..\..\test\include
```

Comment:
Can you add a comment here that this is testing two things? (quotes and also escaped quotes)

Reply:
The first one is testing a space in the path (the quotes are just to allow the path w/ space to pass through NMake to the wrapper as one argument), but yes, happy to.

Reply:
I think I'm on the same page: I don't want to mandate any phrasing, more just that a comment highlight the two different things that are being tested.

Reply:
Updated